components/header.tsx

```diff
@@ -0,0 +1,135 @@
+import Link from "next/link";
+import Button from "@material-ui/core/Button";
+
+const linkStyle = {
```

Comment:
Is there a reason that you use two different styling, styles. cssInJs and otherwise?

Reply:
Grunden til det er at man ikke kan style komponenter fra matrial ui på samme måde.